supermarq-benchmarks/supermarq/converters.py

```diff
@@ -20,7 +20,8 @@ def cirq_to_qiskit(circuit: cirq.Circuit) -> qiskit.circuit.QuantumCircuit:
 def compute_communication_with_qiskit(circuit: qiskit.circuit.QuantumCircuit) -> float:
     """Compute the program communication of the given quantum circuit.
 
-    Program communication = circuit's average qubit degree / degree of a complete graph
+    Note:
+        Program communication = circuit's average qubit degree / degree of a complete graph.
```

Comment:
I thought that formatting was more standard in Google Style Python Docstrings. In one of the examples,

```ruby
def example_method(self, param1, param2):
        """Class methods are similar to regular functions.

        Note:
            Do not include the `self` parameter in the ``Args`` section.

        Args:
            param1: The first parameter.
            param2: The second parameter.

        Returns:
            True if successful, False otherwise.

        """
        return True
```

Reply:
Oh interesting. Basically a nit, but I think we decided to go right into text, without the `Note:`. See [here](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) (this was linked in the server-side dev guide section on docstrings). @richrines1 was this choice deliberate?

Reply:
I don't have a strong preference either way, my thought process in this instance was that using `Note:` would emphasize/highlight the equation specifically (amongst the text). But will revert to follow our dev guide more.